sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -38,6 +38,10 @@ export const ServiceEnvironmentVariable = {
   PLAYWRIGHT_SERVICE_REPORTING_URL: "PLAYWRIGHT_SERVICE_REPORTING_URL",
 };
 
+export const oneTimeEnviromentVariable = {
```

Comment:
Use internal environment variables

Reply:
Also make it generic variable such that any operation which needs to be done 1 time can be done behind this flag